docs/content/components/list.mdx

```diff
@@ -20,27 +25,52 @@ import { Wrapper } from '../../utils/Wrapper';
   </List>
 </Playground>
 
-## Variants
+## When to use it
+
+Use it when you want to display content related to a single subject. 
```

Comment:
Is this definition complete? I feel that something may be missing. 🤔

Reply:
Yes, it was a leftover from the rebase. Will fix it with the correct definition. Thanks!